hw/ip/edn/data/edn.hjson

```diff
@@ -84,24 +84,23 @@
                 software before any further commands can be issued to these FIFOs.
                 '''
         }
-        { bits: "2",
-          name: "AUTO_REQ_MODE",
+        { bits: "3:2",
+          name: "HW_REQ_MODE",
           desc: '''
+                2'b00 - Disables BOOT_REQ_MODE and AUTO_REQ_MODE.
+                2'b01 - AUTO_REQ_MODE:
                 Setting this bit will enable the EDN block to automatically send another request to
                 CSRNG application interface. It is assumed that the command has been set up ahead of
                 time, such that when the previous command has returned ack, then a new command will be
                 send out again without software intervention. It is expected that the generate command
                 will be sent repeatedly so that a continuous supply of entropy can be delivered to the
                 endpoints.
-                '''
-        },
-        { bits: "3",
-          name: "BOOT_REQ_DIS",
-          desc: '''
-                Setting this bit will disable the feature where the EDN block will automatically
+                2'b10 - BOOT_REQ_MODE:
+                This decode will enable the feature where the EDN block will automatically
                 send a boot-time request to the CSRNG application interface.
                 The purpose of this feature is to request entropy as fast as possible after reset,
                 and during chip boot-time.
+                2'b11 - Reserved.
```

Comment:
Looking at the RTL, `2'b11` right now seems to behave just like `2'b00`. Do you want to state this in the documentation? Alternatively, but this will be more work, you could make this register or field `hwext` and then resolve reserved values to `2'b00`. This is what we do for AES.

Reply:
In most documentation that I have seen, a designation of reserved is not specified, mostly for future use. If the reserved field is programmed, it is assumed that unpredictable results could occur. Unless there is some security reason that I am not familiar with for AES, mapping the reserved values seems unneeded.

Reply:
I am just worried about the worst case scenario where an attacker would flip a single bit from `2'b01` to `2'b11` and then EDN would enter an undefined state where it delivers constant "entropy". But it looks like the implementation currently wouldn't return anything / not acking the requests and the consumer would just starve. This is the way to go.